### ISSUE TITLE
How to customize a loss function in pytorch?

### ISSUE THREAD

**Issue by stevenhanjun** [pytorch/pytorch#1352]

**Comment by soumith** [comment]
search discuss.pytorch.org for answer
